framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioImpl.java

```diff
@@ -118,22 +102,22 @@ public SmartRouterDeployment getSmartRouterDeployment() {
 
     @Override
     public KieServerDeployment getKieServerOneDeployment() {
-        return kieServerOneDeployment;
+        return kieServerDeployment;
     }
 
     @Override
     public KieServerDeployment getKieServerTwoDeployment() {
-        return kieServerTwoDeployment;
+        return kieServerDeployment;
```

Comment:
If there is only one Kie Server and Dabatase it should be good to add some logger message with warning when is get second deployment.

Reply:
Added warning. Please take a look